dipy/tracking/utils.py

```diff
@@ -937,3 +939,50 @@ def reduce_rois(rois, include):
             exclude_roi |= rois[i]
 
     return include_roi, exclude_roi
+
+
+def vals_from_img(img, streamlines, affine=None):
+    """
```

Comment:
Also having input an img rather than data and affine is something rarely used in dipy. Do you really need img as input?